web/src/components/core/ServerError.jsx

```diff
@@ -28,19 +28,19 @@ import { locationReload } from "~/utils";
 
 const ErrorIcon = () => <Icon name="error" className="icon-xxxl" />;
 
-function DBusError() {
+function ServerError() {
   return (
     // TRANSLATORS: page title
-    <Page icon="problem" title={_("D-Bus Error")}>
+    <Page icon="problem" title={_("Agama Error")}>
       <Center>
         <EmptyState variant="xl">
           <EmptyStateHeader
-            titleText={_("Cannot connect to D-Bus")}
+            titleText={_("Cannot connect to Agama server")}
             headingLevel="h2"
             icon={<EmptyStateIcon icon={ErrorIcon} />}
           />
           <EmptyStateBody>
-            {_("Could not connect to the D-Bus service. Please, check whether it is running.")}
+            {_("Could not connect to the Agama server. Please, check whether it is running.")}
```

Comment:
I would say this part of the message is not useful at all. The first sentence is basically a repetition of the bigger "Cannot connect to...". And, as a user, how can I check whether the service is running?

By now, I would remove the first sentence. About the second part, we should link to some page explaining how to proceed. But that's out of the scope of this card.

Reply:
I agree